controls/listview/cells/listview-cells.md

```diff
@@ -6,64 +6,66 @@ position: 1
 slug: listview-cells
 ---
 # ListView Cell Types
-Cells are non visual objects used to describe a template for creating a visual element inside each ListView item. There are two types of cells:
+Cells are non-visual objects used to describe a template for creating a visual element inside each ListView item. There are two types of cells:
 
-- **ListViewTextCell**: derives from Xamarin.Forms.**TextCell** and is meant to visualize text and optionally detail text rendered as second row inside list view item. This is the default cell of the RadListView.
-- **ListViewTemplateCell**: derives from Xamarin.Forms.**ViewCell** and is used to present complex data set as RadListView.**ItemTemplate**.
+- **ListViewTextCell** derives from Xamarin.Forms.**TextCell** and displays text. It can optionally render detail text as a second row within a list view item. This is the default cell of the RadListView.
+- **ListViewTemplateCell** derives from Xamarin.Forms.**ViewCell** and used to present complex data sets as RadListView.**ItemTemplate**.
```

Comment:
Can we add more details about ListViewTemplateCell? The definition doesn't seem exhaustive.